### DISCUSSION TITLE
How to pare down to Windows only

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#15243] [Q&A]
Supposing I use
```
Uno Platform App             unoapp                      [C#]        Multi-platform/Uno Platform/Android/iOS/macOS/Mac Catalyst/Windows/Linux/WebAssembly/WinUI
```
to create a new app, and then decide I don't have the skill set for Android/iOS/macOS/Mac Catalyst/Linux/WebAssembly, how to i strip out all the non-Windows items without breaking the build?

**Comment by jeromelaban** [comment]
You can find the dotnet new templates documentation [here](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows). Specifically `dotnet new -h` will help.

We have some more updates coming on this topic to help around that scenario.
